src/utils/transactions.ts

```diff
@@ -188,3 +189,39 @@ export const getTxOrigin = (app?: SafeAppData): string | undefined => {
 
   return origin
 }
+
+/**
+ * Decodes the data of a multiSend tx into its separate txs
+ *
+ * This function is currently not tested as it's only used by jest tests to decode and check created multiSend txs
+ *
+ * @param data tx.data
+ * @returns Array of multiSend MetaTransactionData
+ */
+export const decodeMultiSendTxs = (data: string): MetaTransactionData[] => {
```

Comment:
Nit: can you add a TODO here to replace this once the [`core-sdk` utility](https://github.com/safe-global/safe-core-sdk/pull/342) is merged.